src/views/Item/Item.jsx

```
@@ -353,6 +494,92 @@ const Item = () => {
+                        <div className="w-full mb-4">
+                            <div className="flex flex-wrap border border-gray-300 p-2 rounded-lg">
+                                {getSelectedValues().map((item, index) => (
+                                    <div
+                                        key={index}
+                                        className="border border-gray-150 text-xs rounded-full px-2 py-1 text-sm mr-2 mt-2 mb-2 flex items-center"
+                                    >
+                                        <p className="mr-2">{item.label}</p>
+                                        <button
+                                            onClick={() => removeFilter(item.type)} // Remove the specific filter
+                                            className="text-red-500 hover:text-red-700"
+                                        >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)